aws/resource_aws_resourcegroups_group_test.go

```diff
@@ -138,3 +193,42 @@ JSON
 }
 `, rName, desc, query)
 }
+
+func testAccAWSResourceGroupConfigTags1(rName, desc, query, tag1Key, tag1Value string) string {
+	return fmt.Sprintf(`
+resource "aws_resourcegroups_group" "test" {
+  name        = "%s"
+  description = "%s"
+
+  resource_query {
+    query = <<JSON
+%s
```

Comment:
Nit: We prefer to hardcode values like these into the test configurations so they can more easily be copy-pasted for manual Terraform CLI usage.